### ISSUE TITLE
[Feature] Service to refresh environment variables value

### ISSUE THREAD

**Issue by hcteixeira** [tomaae/homeassistant-mikrotik_router#298]
## Is your feature request related to a problem? Please describe.
After executing a script in Mikrotik (HA button) I need to check de new value of a environment variable, but it only update after several seconds.

## Describe the solution you'd like
A service to refresh values of the environment variables (all of them or a especific one), so I can create a HA script to call script and after refresh the variable value.

**Comment by tomaae** [comment]
No, that would be unnecessary feature creep. But we could pull script and environment endpoints after executing script, that would make sense as long as script execute is blocking. Some scripts can run for long time.
